include/mbedtls/cipher.h

```diff
@@ -507,6 +507,77 @@ static inline const char *mbedtls_cipher_info_get_name(
         return( info->MBEDTLS_PRIVATE(name) );
 }
 
+/**
+ * \brief       This function returns the size of the IV or nonce
+ *              for the cipher info structure, in bytes.
+ *
+ * \param info  The cipher info structure. This may be \c NULL.
+ *
+ * \return      The recommended IV size.
+ * \return      \c 0 for ciphers not using an IV or a nonce.
+ */
+static inline int mbedtls_cipher_info_get_iv_size(
```

Comment:
Why `int`? Surely an IV is an unsigned value that fits in memory, so this should be `size_t`.

The library does use `int` for unsigned values in some places, but for the last few years we've avoided that in new code unless it was absolutely necessary for backward compatibility.

Same with `get_block_size`.

Reply:
I used `int` to match the corresponding `mbedtls_cipher_get_*` functions. I agree that `size_t` makes more sense.